apps/vs-code-designer/src/app/commands/dataMapper/dataMapper.ts

```diff
@@ -143,7 +144,7 @@ export const loadDataMapFileCmd = async (context: IActionContext, uri: Uri) => {
       context.telemetry.properties.result = 'Canceled';
       context.telemetry.properties.missingSourceSchema = 'true';
 
-      DataMapperExt.showError('No source schema file was selected. Aborting load...');
+      ext.showError('No source schema file was selected. Aborting load...');
```

Comment:
NIT: Lets localize the strings that we show in the output channel

Reply:
Localized all strings related to log statements!